### DISCUSSION TITLE
Cannot Get Grids to Render Properly

### DISCUSSION THREAD

**Discussion by onairmarc** [awcodes/filament-tiptap-editor#409] [General]
Hello,

After following the installation instructions, everything seems to work when inside of Filament. However, when I go to render the html output in a blade file, the grid appears stacked with no styling and is not laid out how it was in the editor. I've verified that the CSS classes are present after rebuilding the CSS files using Vite, but the HTML output isn't rendering as expected.

Since the output format is HTML, I've tried `{!! tiptap_converter()->asHTML($content) !!}`, `{!! $content !!}`, and `{{ $content }}`. That last one was just to be sure I wasn't missing anything.

Below are my current theme configurations and screenshots:

vite.config.js:
```js
/*
 * Copyright (c) 2024. Encore Digital Group.
 * All Right Reserved.
 */

import {defineConfig} from 'vite';
import laravel from 'laravel-vite-plugin';

export default defineConfig({
    plugins: [
        laravel({
            input: [
                'resources/css/app.css',
                'resources/js/app.js',
                'resources/css/webcms/theme.css',
            ],
            refresh: true,
        }),
    ],
});
```

tailwind.config.cjs content
```js
module.exports = {
    content: [
        "./resources/**/*.blade.php",
        "./resources/**/*.js",
        "./resources/**/*.vue",
        "./vendor/encoredigitalgroup/**/*.blade.php",
        './vendor/awcodes/filament-tiptap-editor/resources/**/*.blade.php',
    ]
}
```

postcss.config.js
```js
export default {
    plugins: {
        'postcss-import': {},
        'tailwindcss/nesting': {},
        'tailwindcss': {},
        'autoprefixer': {},
    },
};
```

webcms/theme.css
```css
@import '/node_modules/cropperjs/dist/cropper.css';
@import '/node_modules/tippy.js/dist/tippy.css';
@import '/vendor/filament/filament/resources/css/theme.css';
@import '/vendor/awcodes/filament-curator/resources/css/plugin.css';
@import '/vendor/awcodes/filament-tiptap-editor/resources/css/plugin.css';

@config 'tailwind.config.js';
```

webcms/tailwind.config.js
```js
import preset from '../../../vendor/filament/filament/tailwind.config.preset'

export default {
    presets: [preset],
    content: [
        './app/Filament/**/*.php',
        './resources/views/filament/**/*.blade.php',
        './vendor/filament/**/*.blade.php',
        './vendor/awcodes/filament-curator/resources/**/*.blade.php',
        './vendor/awcodes/filament-tiptap-editor/resources/**/*.blade.php',
    ],
}
```

Editor:
![Screenshot_06-13-24_0106PM](https://github.com/awcodes/filament-tiptap-editor/assets/50760632/f9790d12-14f6-413a-8164-860f4bd4e719)

Rendered Blade View:
![Screenshot_06-13-24_0106PM-01](https://github.com/awcodes/filament-tiptap-editor/assets/50760632/c9045fb5-9683-45d0-8cb8-36f5fecdb2b2)

**Comment by awcodes** [comment]
Outside of the editor, it is up to you to style the content. The plugin can't make any assumptions about how you need to have the content look on the frontend.

**Reply by onairmarc** [reply]
Gotcha! Thanks!

**Reply by LA-87** [reply]
If you what to style it same way as in editor you can try this
```
<div class="tiptap-editor">
    <div class="ProseMirror tiptap-content">
        {!! $content !!}
    </div>
</div>
```
